lib/internal/modules/esm/resolve.js

```diff
@@ -24,11 +24,7 @@ const {
 } = primordials;
 const internalFS = require('internal/fs/utils');
 const { NativeModule } = require('internal/bootstrap/loaders');
-const {
-  realpathSync,
-  statSync,
-  Stats,
-} = require('fs');
+const { realpathSync, statSync, Stats } = require('fs');
```

Comment:
why all those lines have reformatted? Could you please restore them?

Reply:
somewhere along the way a formatter most likely did this